include/mapnik/image_impl.hpp

```diff
@@ -99,10 +99,8 @@ image<T>::image(image<T> const& rhs)
       offset_(rhs.offset_),
       scaling_(rhs.scaling_),
       premultiplied_alpha_(rhs.premultiplied_alpha_),
-      painted_(rhs.painted_)
-#ifdef MAPNIK_METRICS
-     ,metrics_(rhs.metrics_)
-#endif
+      painted_(rhs.painted_),
+      metrics_(rhs.metrics_)
```

Comment:
shouldn't this be guarded?

Reply:
Not anymore, the idea is that if compiled without metrics the declaration is so simple that the compiler should be able to basically remove it or reduce its impact to 0.